Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -174,14 +174,35 @@ public extension UIButton {
         states.forEach { setTitle(title, for: $0) }
     }
 
-    /// SwifterSwift: Center align title text and image on UIButton
-    ///
-    /// - Parameter spacing: spacing between UIButton title text and UIButton Image.
-    func centerTextAndImage(spacing: CGFloat) {
-        let insetAmount = spacing / 2
-        imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
-        titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
-        contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
+    /// SwifterSwift: Center align title text and image
```

Comment:
Something is wrong here... probably an issue with merge

Reply:
I'm not sure how did that happen, I just updated the CHANGELOG.md and somehow it got affected the `UIButtonExtensions.swift` 😅. I have now updated the branch. Sorry.

Reply:
No problem @cHaLkdusT :)